sdk/storage/storage-file/src/generated/src/operations/share.ts

```diff
@@ -91,22 +106,31 @@ export class Share {
    * @param options The optional parameters
    * @param callback The callback
    */
-  deleteMethod(options: Models.ShareDeleteMethodOptionalParams, callback: coreHttp.ServiceCallback<void>): void;
-  deleteMethod(options?: Models.ShareDeleteMethodOptionalParams | coreHttp.ServiceCallback<void>, callback?: coreHttp.ServiceCallback<void>): Promise<Models.ShareDeleteResponse> {
+  deleteMethod(
```

Comment:
These are not related to my code fix. But general formatting code changes

Reply:
oh yeah, will revert those formatting. Thanks for catching it